test/basedriver/helpers-specs.js

```diff
@@ -15,4 +23,20 @@ describe('helpers', () => {
       isPackageOrBundle('/path/to/an.apk').should.be.false;
     });
   });
+
+  describe('#unzipFile', () => {
+    it('should unzip a .zip file', async () => {
+      const appContents = await fs.readFile(getFixture('FakeIOSApp.app'));
+      await unzipFile(getFixture('FakeIOSApp.app.zip'));
+      await fs.readFile(getFixture('FakeIOSApp.app')).should.eventually.deep.equal(appContents);
+    });
+
+    it('should unzip a .zip file (force isWindows to be true so we can test the internal zip library)', async () => {
+      const forceWindows = sinon.stub(system, 'isWindows', () => true);
+      const appContents = await fs.readFile(getFixture('FakeIOSApp.app'));
+      await unzipFile(getFixture('FakeIOSApp.app.zip'));
+      await fs.readFile(getFixture('FakeIOSApp.app')).should.eventually.deep.equal(appContents);
```

Comment:
Again.

Maybe I'm misunderstanding how the `unzipFile` function works. Does it overwrite `FakeIOSApp.app`?